main.go

```diff
@@ -130,7 +130,7 @@ func parseLine(line string) string {
 	split := strings.Split(line, "=")
 	key := split[0]
 	value := split[1]
-	if value[0:1] == `"` && value[len(value)-1:] == `"` {
+	if len(value) > 1 && value[0:1] == `"` && value[len(value)-1:] == `"` {
```

Comment:
ok as it is, but getting a little hard to read

Reply:
I'm getting close to just pulling in a library. But this works for now.